charts/external-es-proxy/templates/external-es-proxy-configmap.yaml

```diff
@@ -31,7 +31,21 @@ data:
         proxy_set_header Connection "Keep-Alive";
         proxy_set_header Proxy-Connection "Keep-Alive";
 
-        location ~ ^/ {
+        # The following "location" rules limit airflow interactions to only their indices. Any further
+        # additions should follow this pattern.
+        location ~* /_count$ {
+          rewrite /_count(.*) /{{ include "logging.indexNamePrefix" . }}.$remote_user.*/_count$1 break;
+          {{- if  or .Values.global.customLogging.awsSecretName  .Values.global.customLogging.awsServiceAccountAnnotation .Values.global.customLogging.awsIAMRole }}
+          proxy_pass http://localhost:{{ .Values.service.awsproxy }};
+          {{- else }}
+          access_by_lua_file  /usr/local/openresty/nginx/conf/setenv.lua;
+          proxy_pass {{.Values.global.customLogging.scheme}}://{{.Values.global.customLogging.host}}:{{.Values.global.customLogging.port}};
+          {{- include "external-es-proxy-trustcerts" . | indent 8 }}
+          {{- end }}
+        }
+
+        location ~* /_bulk$ {
+          rewrite /_bulk(.*) /{{ include "logging.indexNamePrefix" . }}.$remote_user.*/_bulk$1 break;
```

Comment:
if this is just copied code, can we put this whole 'nginx.conf` or `http` block into a template and include it in both external and non-external es configmaps ? that would be DRY, and as long as they're going to be the same then that would be helpful.

Reply:
only parts of the changes are copied like rewrite parts

Reply:
then we need to move the the common changes to global helpers.tpl to share  the common code